cloud/amazon/ec2_elb.py

```diff
@@ -19,22 +19,22 @@
 module: ec2_elb
 short_description: De-registers or registers instances from EC2 ELBs
 description:
-  - This module de-registers or registers an AWS EC2 instance from the ELBs
+  - This module de-registers or registers an AWS EC2 instance(s) from the ELBs
     that it belongs to.
-  - Returns fact "ec2_elbs" which is a list of elbs attached to the instance
+  - Returns fact "ec2_elbs" which is a list of elbs attached to the instance(s)
     if state=absent is passed as an argument.
   - Will be marked changed when called only if there are ELBs found to operate on.
 version_added: "1.2"
-author: "John Jarvis (@jarv)"
+author: John Jarvis
 options:
   state:
     description:
-      - register or deregister the instance
+      - register or deregister the instance(s)
     required: true
     choices: ['present', 'absent']
-  instance_id:
+  instance_ids:
```

Comment:
Done, instance_id added as alias of instance_ids in the documentation.
I have also done the other changes requested, I hope it is ok for you.
Don't hesitate to do other feedbacks, as it is the first time I do a pull request, I am not very confortable with the process